src/platform/packages/shared/deeplinks/security/deep_links.ts

```diff
@@ -27,6 +27,7 @@ export enum SecurityPageName {
    */
   cloudSecurityPostureBenchmarks = 'cloud_security_posture-benchmarks',
   cloudSecurityPostureDashboard = 'cloud_security_posture-dashboard',
+  cloudSecurityPostureVulnerabilityDashboard = 'cloud_security_posture-vulnerability_dashboard',
```

Comment:
Is this needed?

Reply:
Yes: this provides TypeScript support for the `link: 'securitySolutionUI:cloud_security_posture-vulnerability_dashboard'` link that is referenced in the static declarations, so it is needed to make type checking to pass.

Reply:
update: now that I'm using the `securityLink` helper, this isn't strictly needed